x-pack/filebeat/module/panw/panos/test/pan_inc_other.log-expected.json

```diff
@@ -62,6 +72,11 @@
             "0.0.0.0",
             "0.0.0.0"
         ],
+        "related.user": [
+            "crusher",
+            "crusher"
```

Comment:
I wish the `append` processor would deduplicate like the `AppendTo` function in beats.